CONTRIBUTING.md

```diff
@@ -180,4 +180,21 @@ make misspell-correction
   - Send staging tag as PR for review.
 - Create a tag `v{version}` on the merged PR and push remote.
 
+## Merging existing ECS conventions
+
+The Elastic Common Schema (ECS) is being merged into OpenTelemetry Semantic
+Conventions per [OTEP 222][otep222]. When adding a semantic convention that
+exists in some form in ECS, consider the following guidelines:
+
+- Prefer using the existing ECS name when possible. In particular:
+  - Do not amend ECS names to follow plural guidelines or other soft 
+    recommendations.
```

Comment:
do we know how many of these there will be? if there's a lot, maybe we should just drop the pluralization guideline completely (#267)?

(not that i'm in favor of dropping the guideline, but if there's a lot of these then it feels like we will be essentially defacto dropping it)

Reply:
I think we can handle the discussion on dropping the specific rule separately. The main motivation for this line is the discussion in #203, but the rule is also meant to apply to the [counter naming rules](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#naming-rules-for-counters-and-updowncounters), the rules listed [here](https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/metrics.md#instrument-naming) and other rules that are not a 'MUST'. In general, the rule I am trying to write down here is "it is okay to ignore a 'SHOULD' rule when integrating existing ECS conventions for the sake of backwards compatibility with ECS".  I think these rules are still useful to have to guide contributions that do not have an equivalent in ECS.

Reply:
> not that i'm in favor of dropping the guideline, but if there's a lot of these then it feels like we will be essentially defacto dropping it

I think having that guideline as a `soft` recommendation still makes sense for any new attributes and where there are no strong arguments against it.

Reply:
@trask This has been reworded in the last iteration, PTAL

Reply:
I am marking this as solved since we rediscussed this on a discussion below.